src/sentry/models/integrationfeature.py

```diff
@@ -31,10 +35,14 @@ def as_str(cls, feature):
             return "integrations-stacktrace-link"
         elif feature == cls.EVENT_HOOKS:
             return "integrations-event-hooks"
+        elif feature == cls.PROJECT_MANAGEMENT:
+            return "integrations-project-managment"
+        elif feature == cls.INCIDENT_MANAGEMENT:
+            return "integrations-incident-management"
 
     @classmethod
     def description(cls, feature, name):
-        if feature == cls.API:
+        if feature in [cls.API, cls.PROJECT_MANAGEMENT, cls.INCIDENT_MANAGEMENT]:
```

Comment:
@jonesphillip @NisanthanNanthakumar I think we should definitely have new text for these new categories. I'm curious why the decision was not to do that.